canarytokens/channel_input_mysql.py

```diff
@@ -38,25 +38,30 @@ def dataReceived(self, data: bytes) -> None:
         try:
             canarydrop, hit = self.handleQuery(src_host, self.buf)
             self.factory.dispatch(canarydrop=canarydrop, token_hit=hit)
-            print(f"alert dispatched for token {canarydrop.canarytoken.value()}: {hit}")
-        except (NoCanarytokenFound, Exception) as e:
+        except (ValueError, NoCanarytokenFound, NoCanarytokenPresent) as e:
+            log.info(f"{e}")
```

Comment:
We should use `.failure()`

Reply:
Not in this case. `.failure()` would end up reporting to the exceptions webhook and all three of these exceptions are not errors, they're cases where we know why we're bombing and that's the correct response. We get a lot of mysql queries with no token in them, and if there is a token but it was deleted we also don't want to error on that.